tests/nlu/featurizers/test_convert_featurizer.py

```diff
@@ -313,10 +305,31 @@ def test_raise_wrong_model_file(
 def test_raise_invalid_path(
     create_or_load_convert_featurizer: Callable[[Dict[Text, Any]], ConveRTFeaturizer]
 ):
-
     component_config = {FEATURIZER_CLASS_ALIAS: "alias", "model_url": "saved_model.pb"}
 
     with pytest.raises(RasaException) as excinfo:
         _ = create_or_load_convert_featurizer(component_config)
 
     assert "neither a valid remote URL nor a local directory" in str(excinfo.value)
+
+
+@pytest.mark.skip_on_windows
+def test_raise_tensorflow_hub_incompatible(
+    create_or_load_convert_featurizer: Callable[[Dict[Text, Any]], ConveRTFeaturizer],
+    monkeypatch: MonkeyPatch,
+):
+    os.environ["PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION"] = "cpp"
```

Comment:
We should use monkeypatch.setenv("PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION", "cpp").